Odata-docs/odatalib/v7/using-utf8jsonwriter-for-better-performance.md

```diff
@@ -39,7 +39,7 @@ builder.AddService<IStreamBasedJsonWriterFactory>(sp => DefaultStreamBasedJsonWr
 
 ### No support for streaming writes
 
-The default `JsonWriter` implements the `IJsonStreamWriter` and `IJsonStreamWriterAsync` interfaces. This interface allows you to pipe data directly from an input stream to the output destination. This may be useful when writing the contents of large binary file into base-64 output without buffering everything in memory first. The new `ODataUtf8JsonWriter` does not yet implement this interface, which means input from a stream will be buffered entirely in memory before being written out to the destination. If your use case requires writing from a stream input, consider using the default `JsonWriter`.
+The default `JsonWriter` implements the `IJsonStreamWriter` and `IJsonStreamWriterAsync` interfaces. This interface allows you to pipe data directly from an input stream to the output destination. This may be useful when writing the contents of large binary file into base-64 output without buffering everything in memory first. The `Utf8JsonWriter`-based writer does not yet implement this interface, which means input from a stream will be buffered entirely in memory before being written out to the destination. If your use case requires writing from a stream input, consider using the default `JsonWriter`. Follow [this issue](https://github.com/OData/odata.net/issues/2422) to find out when this will be resolved.
 
 ### Supported .NET versions
 
```

Comment:
Line below:
"on .NET Core 3.1 and above (i.e. .NET 6 etc.)" => "on .NET Core 3.1 and above"

Reply:
In any case .NET Core 5 applies too

Reply:
I didn't mention .NET 5 because it was already out of support by then. I wasn't sure if someone who's not conversant with the .NET evolution would know that .NET 6+ is in the same line of success as .NET Core 3.1 given that name change from .NET Core to .NET. Maybe I could say .NET Core 3.1 and .NET 5+